### DISCUSSION TITLE
Verzoek: Beschrijvingen toevoegen in commit messages.

### DISCUSSION THREAD

**Discussion by floort** [MinBZK/Algoritmekader#58] [Ideas]
Momenteel zijn commit messages redelijk generiek: "Update bestandsnaam.md".
Is het mogelijk om de leesbaarheid te vergroten om een omschrijving van updates in de commit messages toe te voegen?

**Comment by ruthkoole** [comment]
Hoi @floort . Gaan we naar kijken. We zijn nog bezig een duidelijke instructie te maken van het maken van een commit, pull request en het reviewen daarvan. We zullen dit daarin meenemen.
